Update pgnet alter settings doc (#45225)
supabase/supabase: apps/docs/content/guides/database/extensions/pg_net.mdx

### Alter settings

> `pg_net` settings must be changed at the system level.

Reviewer: ```suggestion
You must change the `pg_net` settings at the system level.
```

What is the "system level"?

Author: "Not at the session level" - let me know if you think of a better way to put it!

> > **Note:** Changing these settings requires superuser privileges. Contact Support to have the required permission granted for the parameter you want to change, e.g.:

Reviewer: ```suggestion
<Admonition type="note">

Changing these settings requires superuser privileges. Contact Support to have the required permission granted for the parameter you want to change, e.g.:
```

```sql
grant alter system on parameter pg_net.ttl to postgres;

> ```

Reviewer: ```suggestion
```

</Admonition>
```

Author: Not sure about this one, why would we remove the section closure?

Reviewer: There was a reason I can't remember now…
